cmd/manager/main.go

```diff
@@ -25,8 +32,9 @@ import (
 
 // Change below variables to serve metrics on different host or port.
 var (
-	metricsHost       = "0.0.0.0"
-	metricsPort int32 = 8383
+	metricsHost               = "0.0.0.0"
+	metricsPort         int32 = 8383
+	operatorMetricsPort int32 = 8686
```

Comment:
(Non related) Do we have plan to do something with those metrics port ?

Reply:
part of `operator-sdk` `0.9.1`

Reply:
we may, there is a plan to do operator metering.